Altis_Life.Altis/stringtable.xml

```diff
@@ -10188,7 +10188,7 @@
         <Key ID="STR_MAR_Cop_Item_Shop">
             <Original>Cop Item Shop</Original>
             <Czech>Cop Item Shop</Czech>
-            <French>Magasin de la Police</French>
+            <French>Marché police</French>
```

Comment:
Less accurate, and it's not really a "marché" (which means "mart, market place")
I'd leave "Boutique de la Police" or "Magasin de la Police"